de-lib/src/main/java/org/iplantc/de/pipelines/client/views/PipelineAppOrderViewImpl.java

```diff
@@ -115,6 +141,21 @@ public PipelineTask getOrderGridSelectedApp() {
         return appOrderGrid.getSelectionModel().getSelectedItem();
     }
 
+    private void setButtonState() {
+        int size = appOrderGrid.getSelectionModel().getSelectedItems().size();
+        if(size > 0) {
+            removeAppBtn.enable();
+            if(appOrderGrid.getStore().size() > 1) {
+                moveDownBtn.enable();
+                moveUpBtn.enable();
+            }
+        } else {
+            removeAppBtn.disable();
+            moveUpBtn.disable();
+            moveDownBtn.disable();
+        }
+    }
```

Comment:
This may just be my own ignorance, but does this mean that `moveDownBtn` and `moveUpBtn` will be enabled any time more than one app is included in the pipeline? If that's the case, should the buttons be disabled unless an app is currently selected? Similarly, should only one of those buttons be enabled if the first or last app in the list is selected?